common/artifacts/000_wc_artifacts.txt

```diff
@@ -838,6 +1282,155 @@ prosthetic_blade_hand = {
 
 	picture = "GFX_inv_weapon_shortblade_29"
 }
+warsong_battle_drums = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes
+		
+	}
+
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+	command_modifier = {
+#		actual_heavy_infantry = 0.05
+#		actual_light_infantry = 0.05
+#		archers = 0.05
+#		light_cavalry = 0.05
+		offence = 0.1
+		morale_offence = 0.1
+	}
+	picture = "GFX_warsong_battle_drums"
+	
+	slot = leader
+}
+scourge_bone_chimes = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes	
+			OR = {
+			is_death_knight_class_trigger = yes
+			is_necromancer_class_trigger = yes
+			is_warlock_class_trigger = yes
+			OR = {
+				religion = death_god
+				religion_group = fel_group
+			}
+		}
+	}
+
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+
+	general_opinion = -5
+	death_god_opinion = 5
+	command_modifier = {
+		morale_defence = 0.1
+		morale_offence = 0.1
+	}
+	picture = "GFX_scourge_bone_chimes"
+	
+	slot = leader
+}
+frost_wyrm_skull_shield = {
+	quality = 4
```

Comment:
Why it's epic? It stopped appearing in the lore after Warcraft 3. I would give it 3 quality.

Reply:
my bad, values are of tier 3, just forgot to change quality-value to 3